modules/exploits/windows/smb/ms17_010_eternalblue.rb

```diff
@@ -678,7 +678,7 @@ def smb_pass
   end
 
   # Returns the value to be passed to SMB clients for
-  # the username. If the user hs not supplied a username
+  # the username. If the user has not supplied an username
```

Comment:
`a username` would be the [usage](https://www.grammar.com/a-vs-an-when-to-use/). Fix it and I'll land the PR.

Reply:
I have made the changes.
Thanks.